SpectralCubeReprojectExample.ipynb

```diff
@@ -0,0 +1,995 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "qualified-subdivision",
+   "metadata": {},
+   "source": [
+    "# Cube Reprojection Tutorial\n",
+    "\n",
+    "This tutorial shows how to take two spectral cubes observed toward the same part of the sky, but different frequencies, and put them onto the same grid.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "higher-console",
+   "metadata": {},
+   "source": [
+    "## Index \n",
+    "\n",
+    " * [Step 1: Download](#Step-1:-Download-the-data)\n",
+    " * [Step 2: Open files, collect metadata](#Step-2:-Load-the-cubes)\n",
+    " * [Step 3: Convert to velocity](#Step-3:-Convert-cubes-from-frequency-to-velocity)\n",
+    " * [Step 4: Spectral Interpolation](#Step-4.-Spectral-Interpolation)\n",
+    " * [Step 5: Spatial Smoothing](#Step-5.-Spatial-Smoothing)\n",
+    " * [Step 6: Reprojection](#Step-6.-Reprojection)\n",
+    " \n",
+    " \n",
+    "In this example, we do spectral smoothing and interpolation (step 4) before spatial smoothing and interpolation (step 5), but if you have a varying-resolution cube (with a different beam size for each channel), you have to do spatial smoothing first."
```

Comment:
Add an extra line with a "for more info see the spectral-cube docs" + link to the docs: spectral-cube.readthedocs.io